docs/source/traits_user_manual/index.rst

```diff
@@ -21,3 +28,42 @@ Indices and tables
 
 * :ref:`genindex`
 * :ref:`search`
+
+
+License
```

Comment:
Would it make sense to move the License stuff above the indices? I think I expect to find the index as the last item on the page, but others may have different expectations.

Reply:
I would have it at the bottom after the index reference, I think.  Certainly that's what I did for TraitsUI.

Reply:
Okay, SGTM